etc/docs/generate-docs.ts

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env ts-node
+
+import { parse, stringify } from '@iarna/toml';
+import * as child_process from 'child_process';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import { promisify } from 'util';
+
+const exec = promisify(child_process.exec);
+
+const RELEASES_TOML_FILE = './template/data/releases.toml';
+const RELEASES_JSON_FILE = './template/static/versions.json';
+
+interface JsonVersionSchema {
+  version: string;
+}
+
+interface VersionSchema {
+  version: string;
+  status: 'current' | 'supported' | 'not-supported';
+  api: string;
+  usesMongoDBManual?: boolean;
+  docs?: string;
+  semvarVersion: string;
+}
+
+interface TomlVersionSchema {
+  current: string;
+  mongodDBManual: string;
+  versions: VersionSchema[]
+}
+
+/**
+ * ATTENTION - EDIT THIS BEFORE RELEASING
+ */
+const NEW_VERSION: VersionSchema = {
+  // version - the display name for the version of the driver on the docs website
+  //  ex: Driver 4.5
+  version: 'Driver 4.5',
+
+  // status - a status for the version.  typically would probably just be 'latest'
+  status: 'current',
```

Comment:
what does this affect? I didn't see a difference in the TOC based on this

Reply:
It's used in the html template - https://github.com/mongodb-js/learn-mongodb-docs/blob/master/site/layouts/partials/releases.html

Reply:
oh, it looks like it gets turned into a css class, probably worth mentioning or enumerating what classes are available/meaningful; I feel like maybe @nbbeeken was using it to control whether the older docs are displayed or collapsed

Reply:
I can't find any css classes that correspond to the statuses we use now and I confirmed on the current hosted version of our docs that these css classes don't actually apply any styles.  I'll remove the code in the template that applies the styles.  

This value is also used to filter out legacy versions of the driver when building the template though.

Reply:
In that case, should we make it an enum of `'current' | 'not-supported'` and/or add validation that it's not an unknown value (to prevent typos)?

Reply:
We don't currently use the supported class but my quick fix to make the version list table get too long isn't ideal, I think making use of these CSS classes to better indicate the status of each version is a future improvement. Is it okay if we keep this and consider making use of it in the next PR or just future work?

I'm thinking background color like [wikipedia uses](https://en.wikipedia.org/wiki/OpenSSL#Major_version_releases)